includes/class-wc-stripe-webhook-handler.php

```
@@ -515,6 +515,204 @@ public function process_webhook_dispute_closed( $notification ) {
+		if ( ! WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled() || ! function_exists( 'wcs_get_subscriptions_for_order' ) ) {
+			return;
+		}
+
+		$subscriptions = wcs_get_subscriptions_for_order( $order );
+
+		if ( empty( $subscriptions ) ) {
+			return;
+		}
+
+		foreach ( $subscriptions as $subscription ) {
+			if ( 'inactive' === $mandate_status && $revocation_reason ) {
```

PHPMD (2.15.0)

[error] The variable $mandate_status is not named in camelCase. (undefined)

(CamelCaseVariableName)